config/config_test.go

```diff
@@ -378,3 +380,23 @@ func TestConfig_IsProFeatureEnabled(t *testing.T) {
 		})
 	}
 }
+
+func TestIfDefaultImagesVersionsAreInSync(t *testing.T) {
+	defaultConfig, err := NewDefaultConfig()
+	assert.NilError(t, err)
+	initContainersTplFilePath := os.Getenv("REPO_ROOT_DIR") + "/chart/templates/_init-containers.tpl"
```

Comment:
Wouldn't this mean that the unit test will only work when run in CI (where this var is set?!)? I guess we also cannot work with `go:embed` because we are outside of the config dir...
As said. Totally fine if we only add a comment. Sorry for the additional effort because of my comment 😅

Reply:
I forgot to push changes in `./hack/test.sh`. How do you usually run the unit tests locally? If it's via `hack/test.sh` then it will work in locally as well as in CI. IMHO it would be more future proof to have a check in place for version sync, hence if it's not an issue from the developer experience POV, I'd keep this test.

Reply:
I sometimes just run `go test` directly (e.g. via `run package tests` in the IDE). Couldn't we just use `../chart/templates/_init-containers.tpl`, so we would not need the env var? Sure, if we move this unit test file or the template we have to adjust this as well.

Reply:
good point, gonna change it to a relative path